src/content/pages/sprints.mdx

```diff
@@ -0,0 +1,126 @@
+---
+title: EuroPython 2024 Sprints
+subtitle: Organise or join a sprint in EuroPython 2024!
+social_card: sprints.png
+---
+
+import { Image } from "astro:assets";
+
+import sprintImage from "./images/sprint_collection_2420x1619.png";
+
+# EuroPython 2024 Sprints
+
+EuroPython 2024 Sprints will be during the weekend 13-14 July. This year, we are
+hosting the sprint weekend at a **different venue** and it will be free for
+anyone to join!
+
+As per our tradition, the conference organisers will provide the rooms and
+facilities but the sprints are organised by **YOU**. It is a great chance to
+contribute to open-source projects large and small, learn from each other, geek
+out and have fun. 🐍
+
+Lunch and coffee will be provided.
+
+**When**: 13-14 July 2024 (09:30 - 17:00)
+
+**Where**: Rajska Building (RB),
+[Žižkov Campus, VŠE](https://www.vse.cz/english/about-vse/profile/campus/)
+(Prague University of Economics and Business)
+
+- **sprint rooms** - all on the 1st floor of RB
+- **chillout area**: the
+  [atrium](https://drive.google.com/file/d/19qKwC7fmxupRINP5Pvj6Npe_eq0czI6Y/view?usp=sharing)
+  on the ground floor of RB
+
+{/*
+TODO: link map / page about sprint venue
+*/}
+
+**Who can attend**:
+
+- If you have an in-person EuroPython ticket
+  ([Conference, Tutorial or Combined](s#ticket-types)), then access to the
+  sprints is already included. Remember to select your sprints attendance when
+  buying / configuring your ticket, to let the organisers know.
+- If you do not have an in-person conference ticket, you can register to join
+  the sprints for free. Registration information will be announced soon
+  [here](tickets#sprint-tickets).
```

Comment:
This section doesn't exist in the tickets page.

Reply:
I removed the sprints part, but left the tickets part.